packages/mdc-top-app-bar/fixed/foundation.ts

```diff
@@ -21,7 +21,6 @@
  * THE SOFTWARE.
  */
 
-import {MDCTopAppBarAdapter} from '../adapter';
```

Comment:
Seems like the coverage score went down, can we aim to have 100% test coverage?

![image](https://user-images.githubusercontent.com/47312/57874331-512f2500-77de-11e9-8e2a-49f126c34ff0.png)